instrumentation/opentelemetry_oban/lib/opentelemetry_oban/plugin_handler.ex

```diff
@@ -1,4 +1,6 @@
 defmodule OpentelemetryOban.PluginHandler do
+  require OpenTelemetry.Tracer, as: Tracer
```

Comment:
We don't use the macros so this can just be aliased.
```suggestion
  alias OpenTelemetry.Tracer
```